production/technical_userguide.md

```diff
@@ -666,3 +666,69 @@ curl https://api.bcda.cms.gov/data/45/99dbc86f-74e5-4553-88a9-af3e718cb72b.ndjso
 The response will be the requested data as [FHIR Coverage resources](https://www.hl7.org/fhir/coverage.html){:target="_blank"} in NDJSON format.
 
 An example of one such resource is available in the [guide to working with BCDA data](/data-guide/#sample-bcda-files).
+
+## Filtering Your Data with `_since`
```

Comment:
I think this PR is for the sandbox UG updates, right?  I'm curious why the production updates are replicated in here, since they are covered in the other PR.